api/src/main/java/org/openmrs/Person.java

```diff
@@ -59,6 +62,9 @@
 	@Field
 	private String gender;
 	
+
+	@Field(analyze = Analyze.YES)
+	@DateBridge(encoding = EncodingType.STRING, resolution = Resolution.DAY)
```

Comment:
@ibacher I changed it to DAY and it is still working as expected. Is this concern of yours then adressed?